airflow/jobs/scheduler_job.py

```diff
@@ -126,7 +127,14 @@ def _run_file_processor(result_channel,
         log = logging.getLogger("airflow.processor")
 
         set_context(log, file_path)
-        setproctitle("airflow scheduler - DagFileProcessor {}".format(file_path))
+
+        if inherited_conf is not None:  # pylint: disable=too-many-nested-blocks
+            for section in inherited_conf:
+                for key, value in inherited_conf[section].items():
+                    if value not in conf:
+                        conf.set(section, key, value.replace("%", "%%"))
+
+                    setproctitle("airflow scheduler - DagFileProcessor {}".format(file_path))
```

Comment:
When is inherited_conf used -- I can't find this one used anywhere either.

We want to set the process title always.

Reply:
Ah, this is just a miss-indentation. I'll fix it.